### ISSUE TITLE
OpenID login redirect blocked by CSP

### ISSUE THREAD

**Issue by nijel** [WeblateOrg/weblate#3048]
**Describe the bug**
Any OpenID based login provider blocks on intermediate page waiting for user to click on "Continue" button.

**To Reproduce**
1. Go to https://l10n.opensuse.org/
2. Click on Login
3. Open browser devel tools
4. See CSP error

**Expected behavior**
The login should redirect automatically.

**Additional context**
This happens on other servers and with other OpenID providers, the server above is just an example.

**Comment by nijel** [comment]
Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
